client/state/cancellation-offers/selectors/is-fetching-cancellation-offers.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Is there a pending request for cancellation offers
+ * Cancellation Offers is keyed on purchase ID
+ */
+export default function isFetchingCancellationOffers( state, purchaseId ) {
+	return state.cancellationOffers?.[ purchaseId ]?.isFetching ?? null;
```

Comment:
I would understand that this function returns a boolean type `true` or `false`; not `true` / `false` or `null`. Especially from the way we handle it in the rest of the code.

Reply:
good call, I'll update this.

Reply:
After some testing, I remember why it is implemented with `null`.
This allows us to check for the initial "has never been fetched" condition as well as the true/false is actively fetching. This allows us to make the fetch once without needing something like `hasAlreadyTriedFetching`.